webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/apidata/JsonQueryObjects.scala

```diff
@@ -359,6 +360,57 @@ object JsonQueryObjects {
       _isEnabled:  Option[Boolean]
   )
 
+  final case class JQGroupCategory private[apidata] (
+      id:          Option[NodeGroupCategoryId] = None,
+      name:        Option[String] = None,
+      description: Option[String] = None,
+      parent:      Option[NodeGroupCategoryId] = None
+  ) {
```

Comment:
simply moved from the old `RestGroupCategory` data type, the handling logic is the same